lib/app/view/app.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
```

Comment:
Remove this file from the pull request. It will create merge conflict. 

State only the profile page